p2p/sentry/sentry_grpc_server.go

```diff
@@ -415,7 +415,7 @@ func runPeer(
 
 			b := make([]byte, msg.Size)
 			if _, err := io.ReadFull(msg.Payload, b); err != nil {
-				logger.Error(fmt.Sprintf("%s: reading msg into bytes: %v", peerID, err))
+				logger.Error(fmt.Sprintf("%s: reading msg into bytes: %v", hex.EncodeToString(peerID[:]), err))
```

Comment:
```suggestion
				logger.Error(fmt.Sprintf("%x: reading msg into bytes: %v", peerID, err))
```

Any reason not to do that ^^ ?

Reply:
nope, they are both the same